taskcluster/app_services_taskgraph/target_tasks.py

```diff
@@ -22,6 +25,20 @@ def target_tasks_pr_skip(full_task_graph, parameters, graph_config):
 def target_tasks_release(full_task_graph, parameters, graph_config):
     return full_task_graph.tasks
 
+@_target_task('nightly')
+def target_tasks_nightly(full_task_graph, parameters, graph_config):
+    try:
+        nightly_json = taskcluster.get_artifact_from_index(
```

Comment:
Any reason not to look for `project.application-services.v2.branch.main.revision.{head_rev}.taskgraph.decision-nightly` instead? (That would be more in line with how other projects handle this)

Reply:
No reason at all, I just didn't realize it was that simple.  Updated the code to do this.